Avenge

LaCumbiaDelCoronavirus opened the pull request:
## About the PR
Removed salt, the ore.

## Why / Balance
Salt, as an ore, has absolutely no use and is a waste of space when generating ores.

Reasons (think of these before you try to justify the existence of salt!):
- Even if I were to increase the price of it, any reasonable new price would still be far worse than what you can get for selling uranium, gold, silver, etc..
- If you suggest a possible use for salt, note that you'll need more than just one thing salt can do other than be turned into salt/iodine in chemistry. Look at uranium, plasma: they're expensive, used in generators, and used in making alot of things. Steel, glass (quartz/coal+iron): used in making/building alot of things. Also worth alot in decent quantities (like the amounts you can get after mining a full asteroid quartz/coal+iron)

## Technical details
removed all mentions of it other than textures. set saltore to null and any salt ore tile to an empty ore tile for map migrations
Textures were kept incase something else wanted to use them

## Media
see: no salt
<img src=https://github.com/user-attachments/assets/cc6ffcc4-df12-492d-8ced-27ed28328b19 width=300>

## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [ ] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [ ] I have added media to this PR or it does not require an ingame showcase.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them.
This will be posted in #codebase-changes. -->

**Changelog**
:cl:
- remove: Removed salt ore.

LaCumbiaDelCoronavirus commented:
Oops thats the branch name

slarticodefast commented:
Previous identical PR:
https://github.com/space-wizards/space-station-14/pull/35541
Back then maintainers decided it should simply be made more rare or get an actual use.
But the author closed the PR instead of addressing feedback.
Also don't completely delete the prototypes.

LaCumbiaDelCoronavirus commented:
> Previous identical PR: #35541 Back then maintainers decided it should simply be made more rare until it gets an actual use. But the author closed the PR instead of addressing feedback. Also don't completely delete the prototypes.

Yeah that PR was the idea of this. Anyway, I don't see the point in keeping it until it gets a use. I feel it's completely reasonable to add it back when it does actually get a use
I will re-add the prototypes, sure

LaCumbiaDelCoronavirus commented:
Fuck it i will actually just add function to salt in new pr
